.github/workflows/test-and-deploy.yml

```diff
@@ -40,8 +40,15 @@ jobs:
           terraform_version: 1.0.0
           terraform_wrapper: false
 
-      - name: Run Unit Tests
-        run: make install test
+      - name: Run make install
+        run: make install
+
+      - name: Run golint
+        if: ${{ matrix.go == '1.16' || matrix.go == '1.17' }}
```

Comment:
Why does this not apply to 1.15?

Reply:
The provider's `go.mod` file specifies version 1.16, so running `go mod tidy` fails on anything lower

Reply:
Since 1.15 is no longer a supported version and given this is still in pilot, I say drop 1.15 from the run matrix, simplify the job to `make install golint test`, and update the README to reflect the version change.